### DISCUSSION TITLE
Invoices- Display activity description

### DISCUSSION THREAD

**Discussion by thrildor** [kimai/kimai#4108] [Questions & Answers] [answered]
Hello,

I would like to display the field activity description within my invoice template. 

I tried it with 

                                    {% if entry['entry.activity'] is not null %}
                                        <span class="entry-activity">({{ entry['entry.activity.comment'] }})</span>
                                    {% endif %}

but as i thought it will not work
I would have tried it with only activity.comment but from where should the SW than know which activity is meant for the current position of the Invoice.

I originall used the "invoice grouped by projects" template and made only some styling changes

thanks in advance

**Comment by kevinpapst** [comment]
That field is not available: https://www.kimai.org/documentation/invoices.html#timesheet-entry-variables

**Reply by thrildor** [reply]
i thought so

is there another solution?
I saw that ${activity.comment} is an available field and its the field i would need

i just dont know how to implement it correctly because as far as is understood, it does not specify from which activity this field is coming from

thanks again for your help

**Comment by thrildor** [comment, marked as answer]
Solution for me was 

                                                {% if invoice['query.activity.comment'] is not empty %}
                                                    <span class="entry-description">{{ invoice['query.activity.comment'] }}</span>
                                                {% endif %}

Than the comment field of the Activity can be used
